### PR TITLE
Custom events API

### PR DESCRIPTION
**Pull request by jsani-radar**
- Custom events API
- Standardize json in API docs to 2 spaces

### CODE REVIEW

File: docs/api.mdx [hunk 1]
```diff
@@ -1804,12 +1804,63 @@ An event represents a change in user state. Events can be uniquely referenced by
...
+#### Create an entry or exit event
 
-To create an event, call the [track](#track) endpoint to trigger an event. For example, specify a `latitude` and `longitude` to enter a geofence that you have created.
+To create an entry or exit event, call the [track](#track) endpoint to trigger an event. For example, specify a `latitude` and `longitude` to enter a geofence that you have created.
 
 On iOS and Android, use the [SDK](/sdk) to track users.
 
+#### Create a custom event
+
+Create a custom event to analyze alongside your entry and exit events. This can represent anything from a conversion or purchase to engagement with an in-app feature.
```
Comment (david-goodfellow) [inline docs/api.mdx:1815]: are we trying to word it as "alongside entry and exit events" or more "alongside location activity"?

File: docs/api.mdx [hunk 2]
```diff
@@ -1804,12 +1804,63 @@ An event represents a change in user state. Events can be uniquely referenced by
...
+
+###### Definition
+
+<span className="badge badge--info">POST</span> <code>https://api.radar.io/v1/events</code>
+
+###### Body parameters
+
+- **`type`** (string, required): The type of the custom event.
+- **`metadata`** (dictionary, optional): The metadata of the custom event.
+- **`userId`** (string, required): The stable unique ID for the user. Used to identify logged in users.
```
Comment (david-goodfellow) [inline docs/api.mdx:1825]: should we mention that it is the customer's unique ID not radar's? Not sure the pattern we use in describing this across the docs

File: docs/api.mdx [hunk 3]
```diff
@@ -1804,12 +1804,63 @@ An event represents a change in user state. Events can be uniquely referenced by
...
+
+Create a custom event to analyze alongside your entry and exit events. This can represent anything from a conversion or purchase to engagement with an in-app feature.
+
+###### Definition
+
+<span className="badge badge--info">POST</span> <code>https://api.radar.io/v1/events</code>
+
+###### Body parameters
+
+- **`type`** (string, required): The type of the custom event.
```
[the remark this answers is not in the document]
Comment (jsani-radar) [reply docs/api.mdx:1823]: good calls all around, added

File: docs/api.mdx (hunk 2 above)
Comment (jsani-radar) [reply docs/api.mdx:1825]: intention was to achieve parity with what we list alongside the identifiers for track - https://radar.com/documentation/api#track

**Comment by andlum** [comment]
Is there a plan for adding the SDK functionality to the docs too?

**Comment by jsani-radar** [comment]
I can take care of that with my existing custom events PR

or perhaps I'll do it in a follow-up to not block on mobile going

**Pull request merged**